src/libraries/System.Text.Encodings.Web/src/System/Runtime/Intrinsics/ArmIntrinsics.Shims.cs

```diff
@@ -0,0 +1,145 @@
+﻿namespace System.Runtime.Intrinsics.Arm
+{
+    internal abstract class ArmBase
```

Comment:
Doing what was done here: https://github.com/dotnet/runtime/pull/38597/files#diff-41fd4c3dcafff7e9e07b5eed514a9e22R18, and just reusing the existing definitions might be better.

Reply:
Cool, I should probably wait until that PR is merged to avoid conflicts.

Reply:
I just merged my PR, so you should be good now :)

Reply:
Oh no, more rebasing :'(

Reply:
I share your pain, @eiriktsarpalis 😄